src/llm_vm/completion/optimize.py

```diff
@@ -99,22 +100,13 @@ def init_if_null(self, c_id):
                                           "model": None }
 
 
-def CALL_BIG(prompt, gpt4=False, **kwargs):
-    print("HERE",os.getenv("OPENAI_API_KEY"))
-    cur_prompt = [{'role': "system", 'content' : prompt}]
-    print("CUR_PROMPT:", cur_prompt, flush=True)
-    print("KWARGS:", kwargs, flush=True)
-    ans = openai.ChatCompletion.create(
-        messages=cur_prompt,
-        model="gpt-3.5-turbo-0301" if not gpt4 else 'gpt-4',
-        **kwargs)
-
-    return ans['choices'][0]['message']['content']
-
-def CALL_SMALL(*args, **kwargs):
-    ans = openai.Completion.create(*args, **kwargs)
-    return ans['choices'][0]['text']
+def CALL_BIG(prompt, **kwargs):
+    model = models.return_generation_big()()
```

Comment:
This creates a new model object from onsite_llm.py every call, not sure if this is a good idea. We should probably store some state so we don't remake this everytime

Reply:
not a good idea, but easy to fix.

Reply:
we roughly speaking want a single static config object thats initialized once

Reply:
yeah, this would be painfully slow cause it'd reload the model for every call, no fun. 

lemme make a MR ontop of your MR